runtime/src/lib.rs

```diff
@@ -328,14 +325,38 @@ impl pallet_uniques::Config for Runtime {
 	type Locker = ();
 }
 
+parameter_types! {
+	pub const BasicDeposit: Balance = deposit(1, 258); // 258 bytes on-chain
+	pub const FieldDeposit: Balance = deposit(0, 66); // 66 bytes on-chain
+	pub const SubAccountDeposit: Balance = deposit(1, 53);
+	pub const MaxAdditionalFields: u32 = 100;
+	pub const MaxRegistrars: u32 = 20;
```

Comment:
```suggestion
	pub const BasicDeposit: Balance = deposit(1, 258); // 258 bytes on-chain
	pub const FieldDeposit: Balance = deposit(0, 66); // 66 bytes on-chain
	pub const SubAccountDeposit: Balance = deposit(1, 53);
	pub const MaxAdditionalFields: u32 = 100;
	pub const MaxRegistrars: u32 = 1;
```
not sure about the despitfees per bytes how to read this ?

Reply:
I saw two example configurations.

https://github.com/paritytech/substrate/blob/a7ba55d3c54b9957c242f659e02f5b5a0f47b3ff/bin/node/runtime/src/lib.rs#L1312-L1314

https://github.com/PureStake/moonbeam/blob/2be5c91596e6a5920aabb587a24fa86b55c9cfd2/runtime/moonbeam/src/lib.rs#L653-L657

They look similar but I'm not sure what's the main difference. Still thinking about it.